resources/views/users/newAcc.blade.php

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>SMDB </title>
+
+    <link rel="stylesheet" href="{{asset('css/app.css')}}">
+    <link rel="stylesheet" href="{{asset('css/login.css')}}">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+</head>
+<body>
+    <div class="container-center">
+        <article class="Center">
+            <h1>Create new Account</h1>
+            <form action="{{route('Welcome')}}">
+                @csrf
+                <label>Name:</label> <br />
+                <input id="first_name" name="first_name" type="text"> <br />
+                <label>Email:</label> <br />
+                <input id="email" name="email" type="email"> <br />
+                <label>Password:</label> <br />
+                <input id="password" name="password" type="password"> <br />
+                <label>Confirm password:</label> <br />
+                <input id="password" name="password" type="password"> <br /><br />
+                <button type="submit" class="SigninBtn">Create Account</button>
+            </form>
+        </article>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
```

Comment:
I think these script tags should be moved into the html head. And then use the `defer` attribute to defer execution until document load.

Reply:
Everything should be fixed to the feedback recieved. the scripts are now in the head, with a defer attribute and every path, and pathname have been fixed as well.